Makefile

```diff
@@ -24,23 +27,32 @@ BOWER=bower_components
 
 # JavaScript source and test specs
 JS=scripts
-SOURCE=$(JS)/variables.js \
-	   $(JS)/utils.js \
-	   $(JS)/ermrest.js \
-	   chaise-config.js \
-	   $(JS)/app.js \
-	   $(JS)/facetsModel.js \
-	   $(JS)/facetsService.js \
-	   $(JS)/controller/ermrestDetailController.js \
-	   $(JS)/controller/ermrestFilterController.js \
-	   $(JS)/controller/ermrestInitController.js \
-	   $(JS)/controller/ermrestLoginController.js \
-	   $(JS)/controller/ermrestLogoutController.js \
-	   $(JS)/controller/ermrestResultsController.js \
-	   $(JS)/controller/ermrestSideBarController.js
-
-# List test specs here
-#SPECS=$(JS)/main_spec.js
+SOURCE=$(JS)/vendor/jquery-latest.min.js \
+					$(JS)/vendor/jquery-ui-tooltip.min.js \
+				  $(JS)/vendor/jquery.nouislider.all.min.js \
+				  $(JS)/vendor/bootstrap.min.js \
+				  $(JS)/vendor/jquery.cookie.js \
+				  $(JS)/vendor/angular.js \
+				  $(JS)/vendor/angular-route.js \
+				  $(JS)/vendor/angular-sanitize.js \
+				  $(JS)/vendor/rzslider.js \
+				  $(JS)/vendor/angular-datepicker.js \
+				  $(JS)/vendor/ng-grid.js \
+				  $(JS)/respond.js \
+					$(JS)/variables.js \
+					$(JS)/utils.js \
+					$(JS)/ermrest.js \
+					chaise-config.js \
+					$(JS)/app.js \
+					$(JS)/facetsModel.js \
+					$(JS)/facetsService.js \
+					$(JS)/controller/ermrestDetailController.js \
+					$(JS)/controller/ermrestFilterController.js \
+					$(JS)/controller/ermrestInitController.js \
+					$(JS)/controller/ermrestLoginController.js \
+					$(JS)/controller/ermrestLogoutController.js \
+					$(JS)/controller/ermrestResultsController.js \
+					$(JS)/controller/ermrestSideBarController.js
 
```

Comment:
good luck. a little googling suggests that there have been issues with Atom indenting of Makefiles. while the one's i've seen are tagged 'Closed' I didn't search enough to find what open issues they might have.